src/Columns/ColumnBlob.h

```diff
@@ -0,0 +1,224 @@
+#pragma once
+
+#include <Columns/IColumn.h>
+#include <Compression/CompressedWriteBuffer.h>
+#include <Compression/ICompressionCodec.h>
+#include <Core/Field.h>
+#include <Formats/NativeWriter.h>
+#include <IO/BufferWithOwnMemory.h>
+#include <IO/ReadBufferFromMemory.h>
+#include <IO/WriteBufferFromVector.h>
+#include <Interpreters/castColumn.h>
+#include <Common/WeakHash.h>
+#include "Compression/CompressedReadBuffer.h"
+#include "Core/ColumnWithTypeAndName.h"
+#include "DataTypes/ObjectUtils.h"
+#include "DataTypes/Serializations/ISerialization.h"
+#include "Formats/NativeReader.h"
+#include "base/defines.h"
+
+#include <Poco/Logger.h>
+#include <Common/logger_useful.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+extern const int LOGICAL_ERROR;
+}
+
+
+class ColumnBlob : public COWHelper<IColumnHelper<ColumnBlob>, ColumnBlob>
```

Comment:
According to our style, Blob should be BLOB, as it is an abbreviation of "Binary Large Object" (but BLOb is not as popular as BLOB).